google-api-client.gemspec

```diff
@@ -18,14 +18,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = %w[lib generated]
 
-  spec.required_ruby_version = '~> 2.0'
+  spec.required_ruby_version = '~> 2.4'
 
   spec.add_runtime_dependency 'representable', '~> 3.0'
   spec.add_runtime_dependency 'retriable', '>= 2.0', '< 4.0'
   spec.add_runtime_dependency 'addressable', '~> 2.5', '>= 2.5.1'
   spec.add_runtime_dependency 'mini_mime', '~> 1.0'
-  spec.add_runtime_dependency 'signet', '~> 0.10'
-  spec.add_runtime_dependency 'googleauth', '>= 0.5', '< 0.10.0'
+  spec.add_runtime_dependency 'signet', '~> 0.12'
```

Comment:
Signet was added as an explicit dependency because this library required a more recent version than was guaranteed by the googleauth version specified (See #709). Now that googleauth 0.10.0 has also updated its dependency on signet, does this library still need to declare an explicit dependency on it?

Reply:
I guess not. I'll remove it.